templates/tab/js/default/src/components/sample/lib/useTeamsFx.js

```diff
@@ -14,6 +14,7 @@ export function useTeamsFx() {
   const [result] = useTeams({});
   const { error, loading } = useData(async () => {
     if (!initialized) {
+      setLogLevel(LogLevel.Error);
```

Comment:
What's the default log level? Would it be better if we set default log level to error and explicitly set log level to debug or info for development environment? So that the client code could be: 
```
if (DEBUG_MODE) {
    setLogLevel(LogLevel.Debug);
}
```
Which makes more sense for me.

Reply:
Thanks, updated.